functions/go/set-enforcement-action/README.md

```diff
@@ -0,0 +1,42 @@
+# set-enforcement-action
+
+## Overview
+
+<!--mdtogo:Short-->
+
+Applies the supplied enforcement action on [policy constraints](https://cloud.google.com/anthos-config-management/docs/concepts/policy-controller#constraints) within a package.
+
+<!--mdtogo-->
+
+Policy Controller [constraint objects](https://cloud.google.com/anthos-config-management/docs/how-to/creating-constraints) enable you to enforce policies for your Kubernetes clusters.
+This function provides a quick way for users to set the [enforcementAction](https://cloud.google.com/anthos-config-management/docs/how-to/auditing-constraints#types_of_enforcement_actions)
+attribute to either `dryrun` for auditing, `warn` for letting the non-compliant resource be applied to the cluster with a warning or `deny` for enforcing the constraints and denying the resource application altogether.
```

Comment:
We should not use `either` since we have 3 options here. Please fix the wording

Reply:
Reworded. Made bullets instead.